tests/conftest.py

```diff
@@ -573,3 +583,60 @@ def autouse_fixtures(
 ) -> None:
     """Fixture to control the order of execution of some of the fixtures"""
     return
+
+
+@pytest.fixture(scope="session")
+def installed_mariadb_operator(admin_client: DynamicClient) -> Generator[None, Any, Any]:
+    operator_ns = Namespace(name="openshift-operators", ensure_exists=True)
+    operator_name = "mariadb-operator"
+
+    mariadb_operator_subscription = Subscription(client=admin_client, namespace=operator_ns.name, name=operator_name)
+
+    if not mariadb_operator_subscription.exists:
+        install_operator(
+            admin_client=admin_client,
+            target_namespaces=["openshift-operators"],
+            name=operator_name,
+            channel="alpha",
+            source="community-operators",
+            operator_namespace=operator_ns.name,
+            timeout=Timeout.TIMEOUT_15MIN,
+            install_plan_approval="Manual",
+            starting_csv=f"{operator_name}.v0.38.1",
+        )
+
+        deployment = Deployment(
+            client=admin_client,
+            namespace=operator_ns.name,
+            name=f"{operator_name}-helm-controller-manager",
+            wait_for_resource=True,
+        )
+        deployment.wait_for_replicas()
+    yield
+    uninstall_operator(
+        admin_client=admin_client, name=operator_name, operator_namespace=operator_ns.name, clean_up_namespace=False
+    )
+
+
+@pytest.fixture(scope="class")
+def mariadb_operator_cr(
+    admin_client: DynamicClient, installed_mariadb_operator: None
+) -> Generator[MariadbOperator, Any, Any]:
+    mariadb_csv: ClusterServiceVersion = get_cluster_service_version(
+        client=admin_client, prefix="mariadb", namespace=OPENSHIFT_OPERATORS
+    )
+    alm_examples: list[dict[str, Any]] = mariadb_csv.get_alm_examples()
+    mariadb_operator_cr_dict: dict[str, Any] = next(
+        example for example in alm_examples if example["kind"] == "MariadbOperator"
+    )
+    mariadb_operator_cr_dict["clusterName"] = "1ce302f6-e674-47fa-85be-875ae295c96c"
```

Comment:
should we use a variable for the clusterName?

Reply:
This is unintentional. I will change this. Thanks for catching!